Description: Set default GtkFileChooser directory for scripts and palettes
 The lua scripts and palettes are installed in /usr/share/fceux/ by this
 package. Make the GtkFileChooser open up the correct directories in question
 by default, improving usability.

Author: Alexander Toresson <alexander.toresson@gmail.com>
Forwarded: not-needed

--- fceux.orig/src/drivers/sdl/gui.cpp
+++ fceux/src/drivers/sdl/gui.cpp
@@ -225,7 +225,7 @@
 	fileChooser = gtk_file_chooser_dialog_new ("Open NES Palette", GTK_WINDOW(MainWindow),
 			GTK_FILE_CHOOSER_ACTION_OPEN, GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL,
 			GTK_STOCK_OPEN, GTK_RESPONSE_ACCEPT, NULL);
-	gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(fileChooser), getcwd(NULL, 0));
+	gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(fileChooser), "/usr/share/fceux/palettes");
 
 	if (gtk_dialog_run (GTK_DIALOG (fileChooser)) ==GTK_RESPONSE_ACCEPT)
 	{
@@ -1509,6 +1509,7 @@
 	const char* last_file;
 	g_config->getOption("SDL.LastLoadLua", &last_file);
 	gtk_file_chooser_set_filename(GTK_FILE_CHOOSER(fileChooser), last_file);
+	gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(fileChooser), "/usr/share/fceux/luaScripts");
 	
 	if (gtk_dialog_run (GTK_DIALOG (fileChooser)) ==GTK_RESPONSE_ACCEPT)
 	{
